payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -492,7 +492,11 @@ constructor(
             isVoucher = false,
             requiresMandate = false,
             hasDelayedSettlement = false,
-            shouldRefreshIfIntentRequiresAction = false,
+            // We are intentionally polling for Twint even though it uses the redirect trampoline.
+            // About 50% of the time, the intent is still in `requires_action` status
+            // after redirecting following a successful payment.
+            // This allows time for the intent to transition to its terminal state.
+            shouldRefreshIfIntentRequiresAction = true,
```

Comment:
ooc how long does this poll for?

Reply:
5 times, and delays 2 seconds each time. This is the same for all polling based LPMs.

Reply:
Based on [this](https://github.com/stripe/stripe-android/blob/401a4b0f1fac644161cb46dddebc69220b0e77ea/payments-core/src/main/java/com/stripe/android/payments/PaymentFlowResultProcessor.kt#L210), I think it's 5 times, delaying for [3 seconds](https://github.com/stripe/stripe-android/blob/8dbcea3a47f33c077bf3fd1774f4c52b4ebaca34/stripe-core/src/main/java/com/stripe/android/core/networking/LinearRetryDelaySupplier.kt#L32) each time.

This is the same for all LPMs on Android.